src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -9,13 +9,15 @@
 public class SelectTestFigureOptionListener implements ActionListener {
 
 	private DriverManager driverManager;
-
 	public SelectTestFigureOptionListener(DriverManager driverManager) {
 		this.driverManager = driverManager;
 	}
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		if(e.getActionCommand().contains("Figure Joe 1"))
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.